app/assets/javascripts/discourse/app/components/relative-time-picker.js

```diff
@@ -14,19 +14,21 @@ export default Component.extend({
 
   @on("init")
   cloneDuration() {
-    let mins = this.durationMinutes;
-    let hours = this.durationHours;
+    const passedAttributes = Object.keys(this.attrs);
```

Comment:
Rather than extracting the keys, you can also use `hasOwn` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn : 

![image](https://user-images.githubusercontent.com/920448/223305083-327f52e1-7608-4f3c-b3eb-f36f30512985.png)